recipes/moodycamel-concurrentqueue/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
+set(CMAKE_CXX_STANDARD 11)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(Threads REQUIRED)
```

Comment:
export it in cpp_info.
```suggestion
```

Reply:
Is this possible? `find_package(Threads)` does quite a bit of target-specific work; should all of that be duplicated?

Reply:
Yes, if you use https://github.com/conan-io/conan-center-index/pull/1052#discussion_r389035996 approach

Reply:
I guess that's close enough for most platforms in use, though I would have liked a way to enable that package for the CMake generator to reuse the work it does.  I'll get that working for my targets of interest and push the result.

Reply:
```python
    def package_info(self):
        if self.settings.os == "Linux":
            self.cpp_info.system_libs = ["pthread"]
```

added to `conanfile.py`.